server/routes/messaging.ts

```diff
@@ -35,10 +37,7 @@ router.route("/api/messaging/coordinator-recruitment-text").get(
       }
     );
     if (!resp.ok) {
-      throw {
-        message: MAKE_ERROR_MESSAGE,
-        status: resp.status,
-      };
+      logger.error(new Error(`${resp.status} ${MAKE_ERROR_MESSAGE}`));
```

Comment:
What's the significance of using backticks here instead of single or double quote?

Reply:
There is no significance and I'll change it as part of the commit to adjust the above comments.

Reply:
Backticks let you interpolate values in JS. But it looks like you changed this to something else anyway